ipfx/qc_features.py

```diff
@@ -33,8 +33,16 @@ def measure_vm(vals):
         return 0, 0
     mean = np.mean(vals)
     rms = np.sqrt(np.mean(np.square(vals-mean)))
-    return mean, rms
-########################################################################
+    return float(mean), float(rms)
+
+
+def measure_vm_delta(mean_start, mean_end):
+
+    if mean_end is not None:
```

Comment:
Just wanted to double check, it looks like `mean_end` can either be an `int` or `None`?

Reply:
It is `float` or `None`